src/graphql/query/teamsGoals.graphql

```diff
@@ -1,5 +1,5 @@
-query GetGoals {
-  getGoals(filters: { audienceType: TEAM, isActive: true}) {
+query GetGoals ($teamId: Int, $limit: Int) {
```

Comment:
Will this query still work if these params are empty? We use this query on `/teams`. I'll merge my PR so you can have the latest changes locally, sorry for conflicts!

Reply:
It's merged